### ISSUE TITLE
MARK III: if user requests usernames too much app freezes

### ISSUE THREAD

**Issue by quinnliu** [Laybium/laybium#26]
- [x] try to figure out what is causing the freeze
- [x] if can't figure out how to make it fast either return more than 1 possible username at a time
     or rate limit it

**Comment by quinnliu** [comment]
- [ ] when client asks for new username, server gives 5 usernames. 
- [ ] client responds with which one is used.

**Comment by quinnliu** [comment]
```
{
    // The tab key will cycle through the settings when first created
    // Visit http://wbond.net/sublime_packages/sftp/settings for help

    // sftp, ftp or ftps
    "type": "sftp",

    "sync_down_on_open": true,
    "sync_same_age": true,

    "host": "ec2-54-166-98-77.compute-1.amazonaws.com",
    "user": "ubuntu",
    //"password": "password",
    //"port": "22",

    "remote_path": "/home/ubuntu/laybium_development_workspace/laybium_server",
    //"file_permissions": "664",
    //"dir_permissions": "775",

    //"extra_list_connections": 0,

    "connect_timeout": 30,
    "sftp_flags": ["-o IdentityFile=/Users/appollo_liu/Documents/workspace/laybium_workspace/laybium_key_pair_develop.pem"]
    //"keepalive": 120,
    //"ftp_passive_mode": true,
    //"ftp_obey_passive_host": false,
    //"ssh_key_file": "~/.ssh/id_rsa",
    //"sftp_flags": ["-F", "/path/to/ssh_config"],

    //"preserve_modification_times": false,
    //"remote_time_offset_in_hours": 0,
    //"remote_encoding": "utf-8",
    //"remote_locale": "C",
    //"allow_config_upload": false,
}
```

**Comment by quinnliu** [comment]
Merged into #71 which has been merged into `develop` :)
